### PR TITLE
Switch pipeline to new review app system

### PR DESCRIPTION
**Pull request by patjouk**
Things that need to happen when the PR is merged:
- [ ] rotate secrets,
- [ ] remove unused env vars from staging: slack webhook, github token
- [ ] add secrets on heroku

ref mozillafoundation/mofo-devops#729

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -936,9 +936,29 @@ While for local development we provide a `sample.env` that you can use as defaul
...
+
+*Note:* This only work for Mo-Fo staff: you will need to manually open a Review App on Heroku for PRs opened by external contributors.
+
+#### Review App for branches
+
+You can manually create a review app for any branch pushed to this repo. It's useful if you want to test your code on Heroku without opening a PR yet.
+
+To create one:
+- log into Heroku.
+- Go to the `foundation-site` pipeline.
```
Comment (cadecairos) [inline README.md]:
```suggestion
- Go to the `network-pulse-api` pipeline.
```

File: README.md [hunk 2]
```diff
@@ -936,9 +936,29 @@ While for local development we provide a `sample.env` that you can use as defaul
...
+- Click on `+ New app` and select the branch you want to use.
+
+The review app slack bot will post a message in the `foundation-site` with links and credentials as soon as the review app is ready.
```
Comment (cadecairos) [inline README.md]:
```suggestion
The review app slack bot will post a message in the `mofo-ra-pulse-api` channel with links and credentials as soon as the review app is ready.
```

File: README.md [hunk 3]
```diff
@@ -936,9 +936,29 @@ While for local development we provide a `sample.env` that you can use as defaul
...
+
+#### Environment variables:
+
+- `GITHUB_TOKEN`: GITHUB API authentication,
+- `SLACK_WEBHOOK_RA`: Webhook to `mofo-ra-foundation`
```
Comment (cadecairos) [inline README.md]:
```suggestion
- `SLACK_WEBHOOK_RA`: Webhook to `mofo-ra-pulse-api`
```

**Comment by patjouk** [comment]
Silly copy-pasting without checking >.< Ready for another review

**Pull request merged**